app/utils/db.server.ts

```diff
@@ -29,7 +29,7 @@ export const prisma = remember('prisma', () => {
 						: e.duration < logThreshold * 1.4
 							? 'redBright'
 							: 'red'
-		const dur = chalk[color](`${e.duration}ms`)
+		const dur = styleText(color, `${e.duration}ms`)
```

Comment:
This is great except that the style text utility does not support the redBright color. So we'll have to come up with something else for that.

Reply:
According to the `ForegroundColors` type (which is the type for text colors), the following are allowed:

```ts
type ForegroundColors =
        | "black"
        | "blackBright"
        | "blue"
        | "blueBright"
        | "cyan"
        | "cyanBright"
        | "gray"
        | "green"
        | "greenBright"
        | "grey"
        | "magenta"
        | "magentaBright"
        | "red"
        | "redBright"
        | "white"
        | "whiteBright"
        | "yellow"
        | "yellowBright";
```

`redBright` appears to be allowed (and I am not seeing any type errors)

Reply:
Thanks for double checking on that!